### PR TITLE
fix: added ctrl + enter to run the tests button

### LINKED ISSUE #45175: never was said that you can use ctrl + enter to run the tests
## Affected page

[javascript curriculum](https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures/)

## Your code

none

```




```

## Expected behavior

The users should be able to read somewhere at the beginning of the curriculum that they can use ctrl + enter to run the tests.

## Screenshots

no screenshots
## System

- Device: [e.g. iPhone6, Laptop]
- OS: [e.g. iOS 14, Windows 10, Ubuntu 20.4]
- Browser: [e.g. chrome, safari]
- Version: [e.g. 22]

### PR DESCRIPTION
**Pull request by HenMoshe**
fix(i18n) : added ctrl + enter to run the tests buttun as asked in #45175

Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #45175

### CODE REVIEW

File: client/i18n/locales/english/translations.json [hunk 1]
```diff
@@ -47,7 +47,7 @@
     "cancel": "Cancel",
     "reset-lesson": "Reset this lesson",
     "run": "Run",
-    "run-test": "Run the Tests",
+    "run-test": "Run the Tests(Ctrl + Enter)",
```
Comment (ShaunSHamilton) [inline client/i18n/locales/english/translations.json:50]:
```suggestion
    "run-test": "Run the Tests (Ctrl + Enter)",
```

**Review by ShaunSHamilton — DISMISSED** [review]
Thank you, for taking the time to contribute.

Just a small edit:

File: client/i18n/locales/english/translations.json (hunk 1 above)
Comment (HenMoshe) [reply client/i18n/locales/english/translations.json]: Commited! Thanks! :)

**Review by raisedadead — APPROVED** [review]

**Review by naomi-lgbt — APPROVED** [review]

**Comment by nhcarrigan** [comment]
Thank you for your contribution to the page! 👍
We are happy to accept these changes and look forward to future contributions. 🎉

**Pull request merged**